api/app/utils/enphase_utils.py

```diff
@@ -0,0 +1,131 @@
+import http.client
+import pandas as pd
+from datetime import datetime, timedelta
+import os
+import xarray as xr
+import base64
+import json
+
+from quartz_solar_forecast.forecasts import forecast_v1_tilt_orientation
+from quartz_solar_forecast.pydantic_models import PVSite
+from quartz_solar_forecast.forecast import predict_tryolabs
+from quartz_solar_forecast.data import get_nwp, process_pv_data
+from quartz_solar_forecast.inverters.enphase import process_enphase_data
+
+def get_enphase_auth_url():
```

Comment:
This is duplicated now I think in this file and the original enphase file? Would be good to double check if other are also duplicated and remove duplications were possible

Reply:
> This is duplicated now I think in this file and the original enphase file? Would be good to double check if other are also duplicated and remove duplications were possible

Thanks for pointing thids out, ive removed it in my recent commit